src/Components/Blazor/Templates/src/content/BlazorHosted-CSharp/BlazorHosted-CSharp.Server/WeatherForecast.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace Company.WebApplication1
```

Comment:
The namespace doesn’t look consistent with other files in this project. Also why does this need to be added - wasn’t this class already there before?

Reply:
Ahh right. It's in the shared project. The odd namespace meant this file wasn't causing type collisions and causing the build to fail. I'll remove it.